src/modules/announcements/components/AnnouncementsList.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import { AnnouncementItem } from "./AnnouncementItem";
+import { Pagination } from "../../../components/Pagination/Pagination";
+import { ANNOUNCEMENTS_LIST } from "../../../constants/announcementsList";
+import styles from "./AnnouncementsList.module.scss";
+
+export const AnnouncementsList = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [postsPerPage] = useState(3);
+
+  const indexOfLastPost = currentPage * postsPerPage;
```

Comment:
You could move it to a helper or even utils function and just import and use it here when you need it. Besides, what if the last page would have only one post but it will be the third page? 3 * 3 = 9 and it won't be true cause the last post index on the third page will be in such situation 7.